docs/source/advanced/multi-modules.mdx

```diff
@@ -118,3 +118,8 @@ The `bidirectional` parameter is experimental and not compatible with <a href="h
 
 </ExperimentalFeature>
 
+## Optional optimizations
+
+In Android projects, to further improve build times you can consider making your schema module a jvm module instead of an Android library. Since apollo-kotlin is multiplatform, and android is capable of depending on jvm-only modules, you can benefit from reducing the amount of work needed to compile the schema module. This is true because jvm modules build faster than Android libraries even with identical source code.  
```

Comment:
```suggestion
In Android projects, to further improve build times you can consider making your schema module a JVM module instead of an Android library. Since Apollo Kotlin is multiplatform, and Android is capable of depending on JVM-only modules, you can benefit from reducing the amount of work needed to compile the schema module. This is true because JVM modules build faster than Android libraries even with identical source code.  
```